tests/test_score_justification_endpoints.py

```diff
@@ -23,11 +24,11 @@ def test_get_scores_justifications_list(self, flask_test_client):
         """
         expected_scores_justifications = [
             {
-                "id": "123e4567-e89b-12d3-a456-426655440003",  # noqa
+                "id": str(SqliteTestDB.score_uuid_2),  # noqa
```

Comment:
An alternative to defining/setting a set of uuids could be to use [DeepDiff](https://pypi.org/project/deepdiff/) helper and just exclude the uuid paths from the diff match (if matching the uuids isn't entirely important). If required, there's the helper using DeepDiff [here](https://github.com/communitiesuk/funding-service-design-assessment-store/blob/main/tests/helpers.py) and [an example of](https://github.com/communitiesuk/funding-service-design-application-store/blob/main/tests/test_routes.py#L109) using regex path excludes

Reply:
Beautiful solution as always - will work on this now :)

Reply:
will do this in a BAU so this ticket can get merged